### DISCUSSION TITLE
Is it possible to prevent auto-install?

### DISCUSSION THREAD

**Discussion by michael-reeves-nc** [jdx/mise#3379] [Q&A] [answered]
We're using a very old version of Ruby for one project, which is not supported on our current machines.  This repo does have the tooling for rbenv, but we use docker containers to manage the support the old Ruby.  However, when I'm in this directory, if I run `ruby -v`, mise tries to install the version of ruby specified, which....will fail.

I'd like to prevent this behavior and manually decide when to install the project Ruby.  If mise supported Gemsets, I'd be all down with automatically loading those, but installing Ruby seems a little much.

Thoughts?

**Comment by jdx** [comment, marked as answer]
why is it reading it? `.ruby-version`? `mise tool ruby` or `mise ls ruby` should tell you. if you search "disable" on this page there should be several ways to disable it https://mise.jdx.dev/configuration/settings.html

**Reply by michael-reeves-nc** [reply]
```zsh
$ mise ls ruby
Tool  Version          Config Source                                Requested
ruby  2.3.8 (missing)  ~/Documents/projects/<project>/.ruby-version 2.3.8
ruby  3.3.6

$ mise tool ruby
Backend:            core:ruby
Installed Versions: 2.3.8 3.3.6
Active Version:     2.3.8
Requested Version:  2.3.8
Config Source:      ~/Documents/projects/<project>/.ruby-version
Tool Options:       [none]
```
I needed to set `not_found_auto_install=false` to prevent this behavior.  I thought maybe `task_run_auto_install=false` would do it, but it didn't.

Thanks for the help

BTW, on that page the link [idomatic version files](https://mise.jdx.dev/configuration/settings.html#idiomatic-version-files) doesn't link to anything other than the same page.
